components/script/dom/webidls/GPU.webidl

```diff
@@ -5,15 +5,17 @@
 // https://gpuweb.github.io/gpuweb/#gpu-interface
 [Exposed=(Window, DedicatedWorker), Pref="dom.webgpu.enabled"]
 interface GPU {
+    // May reject with DOMException
```

Comment:
Can we add a Throws annotation, in that case?

Reply:
This was written because ff has throw here when fails getting global. We do not do that, so I removed this.